src/pages/index.tsx

```diff
@@ -28,6 +26,9 @@ interface Props {
   schedule: VaccinationData[];
 }
 
+const Searchbox = dynamic(() => import('../components/Searchbox'));
```

Comment:
same as `Searchbox` as this component is always rendered

I understand making `VaxLocation` dynamic as if we have empty result, we can skip the render

Reply:
Nice explain @pveyes , do you have any suggestion for me ?